cairo1-run/src/cairo_run.rs

```diff
@@ -334,6 +341,39 @@ fn create_proof_mode_header(builtin_count: i16, return_type_size: i16) -> Vec<In
     ctx.instructions
 }
 
+// Create specific instructions to append the return values to the output segment when not running in proof_mode
+// Call the firt program instruction, appends the return values to the output builtin's memory segment and then returns
```

Comment:
If this is intended to be the documentation, then it should be prefixed with the triple slash `///`.